libs/metamask/src/lib/+state/metamask.service.ts

```diff
@@ -1,30 +1,45 @@
 import { Injectable } from '@angular/core';
 import { providers } from 'ethers'
+import { BehaviorSubject } from 'rxjs';
 
 @Injectable({ providedIn: 'root' })
 export class MetamaskService {
 
-  private account: string = '';
+  private _ethereum: any;
```

Comment:
it is at least a record<string, any>

Reply:
const ethereum = (window as any).ethereum;
It's something in window...